src/app/config.ts

```diff
@@ -34,8 +35,23 @@ const config = convict({
 			env: 'MONGODB_URL',
 		},
 	},
+	RECAPTCHA_SECRET_KEY: {
+		doc: 'Secret Google Recaptcha key',
+		default: '',
+		env: 'RECAPTCHA_SECRET_KEY',
+	},
+	RECAPTCHA_PUBLIC_KEY: {
+		doc: 'Public Google Recaptcha key',
+		default: '',
+		env: 'RECAPTCHA_PUBLIC_KEY',
+	},
+	RECAPTCHA_LANGUAGE: {
+		doc: 'Google Recaptcha language',
+		default: 'no',
+		env: 'RECAPTCHA_LANGUAGE',
+	},
 });
-
+config.loadFile(path.join(rootPath, 'config.json5'));
```

Comment:
Check for the existence of that file first, it might not exist at all on a production system where all settings come in through environment variables.